manifests/cf-manifest/deployments/030-cf-jobs.yml

```diff
@@ -1,6 +1,9 @@
 meta:
   environment: ~
 
+  api_domain: (( concat "api." properties.domain ))
+  apps_domain: (( concat "apps." properties.domain ))
```

Comment:
It all depends. Maybe you want to offload cert management to app owners. Last time I have checked certificates with 2 * records were non trivial. Our solution was to use asterisk for apps and then add SAN for api, uaa, login, loggregator etc.